server/routes/visits.js

```diff
@@ -87,7 +89,7 @@ module.exports = [{
       var p = request.payload;
       var fxaId = request.auth.credentials;
       var visitId = p.visitId || uuid.v4();
-      db.createVisit(fxaId, visitId, p.visitedAt, p.url, p.title, function (err, visit) {
+      visit.create(fxaId, visitId, p.visitedAt, p.url, p.title, function (err, visit) {
```

Comment:
`visit` is defined but never used.

Not sure if the return value name of `visit` is confusing since we're already using `visit` as a global alias for the `models.visit`.